### ISSUE TITLE
Oclif command scaffold has outdated license

### ISSUE THREAD

**Issue by joshwlewis** [oclif/oclif#240]
Do you want to request a *feature* or report a *bug*?
-----------------------------------------------------
*bug*

What is the current behavior?
-----------------------------
`npx oclif plugin myplugin` yields a project with a 2018 license.

```bash
$ npx --version
6.4.1
$ node --version
v10.15.3
$ npx oclif plugin myplugin
...
$ cat LICENSE | grep Copyright
Copyright (c) 2018 @joshwlewis
```

What is the expected behavior?
------------------------------
`npx oclif plugin myplugin` yields a project with a 2019 license.

**Comment by jdxcode** [comment]
[The issue is that it's using windows line endings in the travis file](https://github.com/janpio/oclif-ci-demo/blob/master/.travis.yml#L8)

**Comment by RasPhilCo** [comment]
Fixed via #241
